playwright/utils/simple-payment.ts

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-warning-comments */
+import { type Page } from '@playwright/test';
+
+import { selectWallet } from './common.ts';
+
+export const validationMessages = {
+  title: {
+    maxLengthExceeded: 'Title must not exceed 60 characters',
+    isRequired: 'Title is required',
+  },
+  allRequiredFields: [
+    'decisionMethod must be defined',
+    // TODO: Uncomment once issue #38102 is fixed
```

Comment:
The issue number is actually #3802 - can you leave a note on the issue to re-enable this as part of the issue?